README.md

```diff
@@ -0,0 +1,33 @@
+
+# REAP ![Badge Version] [![Badge License]][License] [![Badge Author]][Author]
+
+*A **Library** to provide easy access to* ***ProcFS***
```

Comment:
If you revert this line to what it was before, I'll merge this PR.

Reply:
`this line`, line 2 or 4?
I'm guessing 4?

Reply:
Yeah, 4.

Reply:
Well I took the content of the `Overview` section as the description,
and at the original position there was the table which is now two
badges.

So do you just want the original wording of the description?

Reply:
Yeah, "The REAP library provides Ridiculously Easy Access to Procfs."